### PR TITLE
błagam o approve

### PR DESCRIPTION
**Pull request by brzezin0**

### CODE REVIEW

File: SpokojDucha/components/SearchDecedent.js
Comment (JuliaTrzebuchowska) [inline SpokojDucha/components/SearchDecedent.js:1]: nie bałdzo @WiktoriaKrzyzanska ty popatrz na to tez

**Review by WiktoriaKrzyzanska — CHANGES_REQUESTED** [review]
The add decedent isn't changed as it should for this task. The enetered data isn't visible. Moreover, the "longer text" like a placeholder should be slide-able. The CSS for the placholder is now wrong in many places. The content is missing the mergins that were there before. The search isn't working.  You deleted half of the code needed for the SearchDecedent

File: SpokojDucha/components/SearchDecedent.js [hunk 1]
```diff
@@ -15,12 +15,6 @@ const SearchDecedent = () => {
   const [selectedImage, setSelectedImage] = useState(null);
   const navigation = useNavigation();
 
-  const formatDate = (dateArray) => {
```
Comment (WiktoriaKrzyzanska) [inline SpokojDucha/components/SearchDecedent.js:18]: Why we suddenly don't have this function? If you delete someone's code, give a reason. I don't see a reason. The code is needed for the handling of displaying data in correct format

File: SpokojDucha/components/SearchDecedent.js [hunk 2]
```diff
@@ -64,98 +58,29 @@ const SearchDecedent = () => {
     }
   };
 
-  const handleNavigate = (id) => {
```
Comment (WiktoriaKrzyzanska) [inline SpokojDucha/components/SearchDecedent.js:67]: Anothere delete that never should happen. Not only you deleted the navigation ability but also you deleted the rendering of data that we get from backend

File: SpokojDucha/components/SearchDecedent.js [hunk 3]
```diff
@@ -64,98 +58,29 @@ const SearchDecedent = () => {
...
         placeholder="City"
-        style={[styles.input, { fontSize: 16 + fontSizeDelta }]}
+        style={[styles.input, { fontSize: 16 + fontSizeDelta, paddingVertical: 10 + fontSizeDelta * 0.25 }]}
         onChangeText={value => handleInputChange('city', value)}
         value={keywords.city}
       />
       <TouchableOpacity style={styles.searchButton} onPress={handleSearch}>
         <Text style={[styles.buttonText, { fontSize: 16 + fontSizeDelta }]}>Szukaj</Text>
       </TouchableOpacity>
-      {results.length > 0 && (
```
Comment (WiktoriaKrzyzanska) [inline SpokojDucha/components/SearchDecedent.js:134]: Another part responsible for the search handling. And also needed for the Model that was made our objective for this sprint

File: SpokojDucha/components/SearchDecedent.js [hunk 4]
```diff
@@ -184,54 +108,8 @@ const styles = StyleSheet.create({
   },
   buttonText: {
     color: '#fff',
-    fontSize: 16,
     fontWeight: 'bold',
   },
-  resultsContainer: {
```
Comment (WiktoriaKrzyzanska) [inline SpokojDucha/components/SearchDecedent.js:190]: Why? Why delete so much code once again?

**Review by WiktoriaKrzyzanska — CHANGES_REQUESTED** [review]

**Pull request closed without merging**